### PR TITLE
#167519929 create homepage footer components

### PR DESCRIPTION
**Pull request by Eazybee**
#### What does this PR do?

- This PR create homepage footer components

#### Description of task to be completed?
- create footer molecules
- create footer organisms
- write tests for footers
- write media queries for FlexContainer

#### How should this be manually tested?

  - $ git clone https://github.com/andela/king-kong-ah-frontend.git
  - $ npm install
  - $ npm test

#### Any background context you want to provide?

- You would need npm to run this application

#### What are the relevant pivotal tracker stories?

- PT Story link - [#167519929](https://www.pivotaltracker.com/story/show/167519929)

#### Screenshots (if appropriate)
n/a

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,9 +2,23 @@
 const express = require('express');
 const cors = require('cors');
 const path = require('path');
+const proxy = require('http-proxy-middleware');
 
 const app = express();
-app.use(cors());
+
+const corsOptions = {
```
Comment (benfluleck) [inline app.js]: Do we need these cors options now we have the proxy ?

File: src/components/UI/atoms/Icon/Icon.jsx [hunk 2]
```diff
@@ -89,6 +92,32 @@ display: ${display};
 margin: ${theme.spacing[margin]};
 margin-top: ${theme.spacing[marginTop]};
 margin-left: ${theme.spacing[marginLeft]};
+margin-right: ${theme.spacing[marginRight]};
+
+@media ${theme.device.mobileS} {
+  width: 1.3rem;
+  height: 1.3rem;
+  margin-left: ${marginLeft ? theme.spacing.xxxs : 0};
```
Comment (benfluleck) [inline src/components/UI/atoms/Icon/Icon.jsx:100]: Change to `margin-left: ${marginLeft && theme.spacing.xxxs};`

File: src/components/UI/atoms/Icon/Icon.jsx [hunk 3]
```diff
@@ -89,6 +92,32 @@ display: ${display};
...
+  margin-right: ${marginRight ? theme.spacing.xxxs : 0};
+}
+@media ${theme.device.mobileM} {
+  width: 1.5rem;
+  height: 1.5rem;
+  margin-left: ${marginLeft ? theme.spacing.xxxs : 0};
```
Comment (benfluleck) [inline src/components/UI/atoms/Icon/Icon.jsx:106]: Please keep with the convention above

File: src/components/UI/atoms/RatingsCount/RatingsCount.spec.js [hunk 4]
```diff
@@ -10,7 +10,9 @@ afterEach(cleanup);
 const setup = (color) => {
   const utils = render(
     <RatingsCount
-      color={color ? 'darkPurple' : 'pink'}>
+      color={color ? 'darkPurple' : 'pink'}
```
Comment (benfluleck) [inline src/components/UI/atoms/RatingsCount/RatingsCount.spec.js]: I don't understand why there is conditionally logic to render the color shouldn't we just assign a color

File: src/components/UI/atoms/RatingsCount/RatingsCount.spec.js [hunk 5]
```diff
@@ -10,7 +10,9 @@ afterEach(cleanup);
...
+      counter={100}
```
Comment (benfluleck) [inline src/components/UI/atoms/RatingsCount/RatingsCount.spec.js]: Use default props to set this, please

File: src/components/UI/molecules/Footer/About/About.jsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Title from '<atoms>/Title/Title';
+import Text from '<atoms>/Text/Text';
+
+const About = ({ title, texts }) => (
+  <FlexContainer
+    backgroundColor="primary"
+    padding="zero"
+  >
+    <Fragment>
```
Comment (benfluleck) [inline src/components/UI/molecules/Footer/About/About.jsx:12]: Why do you have a Fragment here when everything is nested in the `TextContainer`

File: src/components/UI/molecules/Footer/About/About.jsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Title from '<atoms>/Title/Title';
+import Text from '<atoms>/Text/Text';
+
+const About = ({ title, texts }) => (
```
Comment (benfluleck) [inline src/components/UI/molecules/Footer/About/About.jsx]: What are texts do you mean textContent

**Review by benfluleck — CHANGES_REQUESTED** [review]
Left feedback

File: src/components/UI/atoms/Icon/Icon.jsx (hunk 2 above)
Comment (Eazybee) [reply src/components/UI/atoms/Icon/Icon.jsx:100]: && returns `false` if the first parameter is undefined;
we will now have something like `margin-left: false` or `margin-left: undefined`

**Review by benfluleck — APPROVED** [review]

**Pull request merged**